workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/ActionNode.java

```diff
@@ -67,7 +67,7 @@
 
     private ProcessInfo processInfo = new ProcessInfo();
     private List<GenericActionNode> containingNodes = new ArrayList<GenericActionNode>();
-
+private int mySonarTest=0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "mySonarTest" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:8081/coding_rules#rule_key=squid%3AS1068)